tests/utils/dockercli/volumelifecycle.go

```diff
@@ -155,3 +155,21 @@ func IsContainerExist(ip, containerName string) bool {
 	}
 	return false
 }
+
+// StopAllContainers - stops all the containers on a particular vm
+func StopAllContainers(ip string) (string, error) {
+	log.Printf("Stopping all containers on VM [%s]\n", ip)
+	return ssh.InvokeCommand(ip, dockercli.StopAllContainers)
+}
+
+// RemoveAllContainers - removes all the containers on a particular vm
+func RemoveAllContainers(ip string) (string, error) {
+	log.Printf("Removing all containers on VM [%s]\n", ip)
+	return ssh.InvokeCommand(ip, dockercli.RemoveAllContainers)
+}
+
+// DeleteAllDockerVolumes - removes all the containers on a particular vm
+func DeleteAllDockerVolumes(ip string) (string, error) {
```

Comment:
it looks cool but dangerous .. it may delete unwanted volumes as well. one serious problem I can think of running test in parallel. 

Ideal approach should be deleting volumes those are created as part of current running test instance.